File: input/instance_impl.zig
pub fn get_device_proc_addr(
    self: InstanceCommands,
    device: Device,
    p_name: [*:0]const u8,
) MissingFunctionError!FpVoidFunction {
    return self.fp_get_device_proc_addr(device, p_name) orelse return error.MissingFunction;
}

pub const CreateDeviceOrMissingFunctionError = CreateDeviceError || MissingFunctionError;
pub fn create_device_to_commands(
    self: InstanceCommands,
    globals: GlobalCommands,
    physical_device: PhysicalDevice,
    p_create_info: *const DeviceCreateInfo,
    p_allocator: ?*const AllocationCallbacks,
) CreateDeviceOrMissingFunctionError!DeviceCommands {
    const device = try self.create_device(physical_device, p_create_info, p_allocator);
    return DeviceCommands.init(globals, self, device, p_create_info);
}

File: input/instance_impl.rs
    pub unsafe fn create_device_commands(
        &self,
        globals: &Globals,
        physical_device: vk::PhysicalDevice,
        p_create_info: &vk::DeviceCreateInfo,
        p_allocator: Option<&vk::AllocationCallbacks>,
    ) -> LoadResult<Device> {
        let device = self.create_device(physical_device, p_create_info, p_allocator)?;
        Device::load(globals, self, device, p_create_info)
    }